cloud/deploys/dev_azure/README.md

```diff
@@ -4,8 +4,42 @@ app. We intentionally don't use a shared state with this terraform directory
 to allow for everyone to have their own app. 
 
 The process to deploy is similar
-to the staging process but we do not have to do the shared state set up. It 
-should be `terraform init` and then `terraform apply`
+to the staging process but run from the cloud/deploys/dev_azure folder. 
```

Comment:
question (blocking): since we're moving staging to using the deployment repo, is this statement still accurate?

Reply:
i guess i meant like we it's the same script minus the wrapper from the deploy repo